dev/bin/cluster-stop.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+########
+# Create and configure a new minikube cluster
+#
+# Env vars:
+# - KIND_NAME: set the current minikube profile (optional, default none)
+# - KIND_CONFIG_FILE: kind config file (https://kind.sigs.k8s.io/docs/user/quick-start/#configuring-your-kind-cluster) (optional. Default is `dev/config/kind/config.yaml`)
+# - KIND_CONTAINER_ENGINE: container engine to use (optional, default is docker)
+# - KIND_NETWORK: If you want to use a different network from the default Kind one (optional)
+# - KIND_KUBERNETES_VERSION: Kubernetes version to use (optional, default="v1.20.0")
+########
+
+. $(dirname "${BASH_SOURCE[0]}")/common.sh
+
+configure_cluster
+
+if [ "${cluster_type}" = 'minikube' ]; then
+    echo "Stopping Minikube"    
+    ${dev_bin_dir}/cluster-minikube-stop.sh
+elif [ "${cluster_type}" = 'kind' ]; then
+    echo "Stopping Kind"
+    ${dev_bin_dir}/cluster-kind-stop.sh
+else
```

Comment:
This is not strictly related with this PR but I think we should decide if we test kind or minikube for local development and align CI, current kind + minikube scripting situation has an overhead that we can probably get rid of.

@robpblake 
Wdyt?

Reply:
@danielezonca Yes agreed. A single approach with either KIND or Minikube makes the most sense. I don't have a strong opinion either way - both should work just fine - but we don't have capacity to support both.

Reply:
+1 to point, the amount of introduced scripts increases the maintain costs. Assuming @radtriste is moving to other project after this is finished. I vote for having single option.